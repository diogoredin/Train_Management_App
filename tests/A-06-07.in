6
3
1
Sapataria
Caldas da Rainha
2017-12-25
06:00
0
0
0
