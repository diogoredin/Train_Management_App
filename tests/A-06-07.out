Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): 
Itinerário 1 para 2017-12-25 @ 4.75
Serviço #6403 @ 4.75
06:52 Sapataria
06:55 Pero Negro
06:58 Zibreira
07:01 Feliteira
07:05 Dois Portos
07:10 Runa
07:15 Torres Vedras
07:23 Ramalhal
07:30 Outeiro
07:41 Bombarral
07:46 Paúl
07:50 São Mamede
07:54 Dagorda-Peniche
07:57 Óbidos
08:03 Caldas da Rainha
Número do itinerário: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: 